app/controllers/mailgun_webhooks_controller.rb

```diff
@@ -22,10 +22,12 @@ def create_incoming_email
       else
         DatadogApi.increment("mailgun.incoming_emails.client_not_found")
 
-        IntercomService.create_intercom_message(
+        IntercomService.create_message(
+          client: nil,
+          phone_number: nil,
           email_address: sender_email,
-          inform_of_handoff: false,
-          body: params["stripped-text"] || params["body-plain"]
+          body: params["stripped-text"] || params["body-plain"],
+          has_documents: false
```

Comment:
Arguably it'd be nice if we validated that `has_documents` is `true` when it should be.

Reply:
After a bit of investigation, it looks there's a _chance_ people can send attachments but we wouldn't be saving them at this time, so `has_documents` can be false